src/System.IO.Packaging/src/System/IO/Packaging/ZipPackage.cs

```diff
@@ -100,21 +183,21 @@ public sealed class ZipPackage : Package
         /// <returns></returns>
         protected override PackagePart GetPartCore(Uri partUri)
         {
-            //Currently the design has two aspects which makes it possible to return
-            //a null from this method -
-            //  1. All the parts are loaded at Package.Open time and as such, this 
+            // Currently the design has two aspects which makes it possible to return
+            // a null from this method -
+            //   1. All the parts are loaded at Package.Open time and as such, this 
             //     method would not be invoked, unless the user is asking for -
             //     i. a part that does not exist - we can safely return null
             //     ii.a part(interleaved/non-interleaved) that was added to the 
             //        underlying package by some other means, and the user wants to 
             //        access the updated part. This is currently not possible as the
             //        underlying zip i/o layer does not allow for FileShare.ReadWrite. 
-            //  2. Also, its not a straighforward task to determine if a new part was 
+            //   2. Also, its not a straighforward task to determine if a new part was 
             //     added as we need to look for atomic as well as interleaved parts and 
             //     this has to be done in a case sensitive manner. So, effectively
             //     we will have to go through the entire list of zip items to determine
             //     if there are any updates. 
-            //  If ever the design changes, then this method must be updated accordingly
+            // If ever the design changes, then this method must be updated accordingly
```

Comment:
Do you mean that we replace the body of the method with something like `Debug.Fail("Never call this message")`